python/lsst/analysis/tools/actions/plot/propertyMapPlot.py

```diff
@@ -41,6 +45,116 @@
+        * ``"lat_max"``
+            Maximum latitude of the original extent (`float`).
+
+    n : `float`, optional
+        Zoom factor; for instance, n=2 means zooming in 2 times at the
+        center. If None, the function returns None.
+
+    Returns
+    -------
+    `tuple` [`float`]
+        New extent as (new_lon_min, new_lon_max, new_lat_min, new_lat_max).
+    """
+    if n is None:
+        return None
+    lon_min, lon_max, lat_min, lat_max = fullExtent
+    lon_center, lat_center = (lon_min + lon_max) / 2, (lat_min + lat_max) / 2
+    half_lon = (lon_max - lon_min) * np.cos(np.radians(lat_center)) / (2 * n)
+    half_lat = (lat_max - lat_min) / (2 * n)
+    return lon_center - half_lon, lon_center + half_lon, lat_center - half_lat, lat_center + half_lat
+
+
+def getLongestSuffixMatch(s, options):
+    """Find the longest suffix in the provided list that matches the end of
+    the given string.
+
+    Parameters
+    ----------
+    s : `str`
+        The target string for which we want to find a matching suffix.
+    options : `list` [`str`]
+        A list of potential suffix strings to match against the target
+        string `s`.
+
+    Returns
+    -------
+    `str`
+        The longest matching suffix from the `options` list. If no match is
+        found, returns `None`.
+    """
+    return next((opt for opt in sorted(options, key=len, reverse=True) if s.endswith(opt)), None)
+
+
+def addTextToColorbar(
+    cb, text, orientation="vertical", color="black", fontsize=14, fontweight="bold", alpha=0.8
+):
+    """Helper method to add text inside the horizontal colorbar.
+
+    Parameters
+    ----------
+    cb : `~matplotlib.colorbar.Colorbar`
+        The colorbar object.
+    text : `str`
+        The text to add.
+    orientation : `str`, optional
+        The orientation of the colorbar. Can be either "vertical" or
+        "horizontal".
+    fontsize : `int`, optional
+        The fontsize of the text.
+    fontweight : `str`, optional
+        The fontweight of the text.
+    alpha : `float`, optional
+        The alpha value of the text.
+
+    Returns
+    -------
+    `None`
+        The text is added to the colorbar in place.
+    """
+    if color is None:
+        color = "black"
+    vmid = (cb.vmin + cb.vmax) / 2
+    positions = {"vertical": (0.5, vmid), "horizontal": (vmid, 0.5)}
+    cbtext = cb.ax.text(
+        *positions[orientation],
+        text,
+        color=color,
+        va="center",
+        ha="center",
+        fontsize=fontsize,
+        fontweight=fontweight,
+        rotation=orientation,
+        alpha=alpha,
+    )
+    # Add a distinct outline around the text for better visibility in
+    # various backgrounds.
+    cbtext.set_path_effects(
+        [mpl_path_effects.Stroke(linewidth=4, foreground="white", alpha=0.8), mpl_path_effects.Normal()]
+    )
+
+
+def prettyPrintFloat(n):
+    if n.is_integer():
+        return str(int(n))
+    return str(n)
+
+
```

Comment:
Here, I simply moved pre-existing static methods outside the class to make them easier to reuse for other tasks in the future.